pkg/controller/nodegroup/nodegroup_controller.go

```diff
@@ -218,7 +222,27 @@ func (r *ReconcileNodeGroup) Reconcile(request reconcile.Request) (reconcile.Res
 		return reconcile.Result{}, err
 	}
 
-	// TODO: Add parameters to obj, and update if parameters change.
+	cfnParameters, err := getCFNParametersFromCFNStack(cfnSvc, instance)
+	if err != nil {
+		r.fail(instance, "error trying to read the CFN Parameters", err, logger)
+		return reconcile.Result{}, err
+	}
+
+	if shouldUpdate(crdParameters, cfnParameters) {
+		logger.Info("Updating the NodeGroup stack with new parameters")
+		err := r.updateNodeGroupStack(cfnSvc, instance, eksCluster, crdParameters)
+		if err != nil {
+			r.fail(instance, "error updating nodegroup cloudformation stack", err, logger)
+			return reconcile.Result{}, err
+		}
+		logger.Info("cloudformation stack updated successfully")
```

Comment:
Are waiting to see the StackUpdate completed in updateNodeGroupStack()?
If not then this log message is misleading.

Reply:
Agreed, have removed the log message